src/index.html

```diff
@@ -9,6 +9,141 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
+<h1 class="">Universal Declaration of Human Rights</h1>
```

Comment:
provide class name, or don't use class at all
```suggestion
<h1>Universal Declaration of Human Rights</h1>
```